ChangeLog.md

```diff
@@ -18,6 +18,10 @@ to browse the changes between the tags.
 
 See docs/process.md for more on how version tagging works.
 
+3.1.14
```

Comment:
The `3.2.0` below should be moved up to replace `3.1.14` here (as this will be in 3.2.0, and the list of versions must be in order).